### ISSUE TITLE
[BUG] Bloxstrap not launching after downloading from Github

### ISSUE THREAD

**Issue by fiull** [pizzaboxer/bloxstrap#983]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

i installed bloxstrap from the github page, after i launched it, it showed me that warning thingy and i pressed more info then "run anyways". But after that nothing happens, i tried opening it again muliple times, then as administrator. I checked my .NET runtime version and i had the latest one. Idk why nothing happens so can someone help? (i also tried to install the winget version but it the bar was stuck at 0%)

**Comment by pizzaboxer** [comment]
Does a log file get produced in %temp%? Sort by date modified, should be prefixed with the name "Bloxstrap_".

**Comment by fiull** [comment]
> Does a log file get produced in %temp%? Sort by date modified, should be prefixed with the name "Bloxstrap_".

Nope, nothing appears. It just shows the loading icon on my mouse for a split second and thats it.

**Comment by Jeseo12** [comment]
> > Does a log file get produced in %temp%? Sort by date modified, should be prefixed with the name "Bloxstrap_".
> 
> Nope, nothing appears. It just shows the loading icon on my mouse for a split second and thats it.
its happens to me too

**Comment by pizzaboxer** [comment]
> Nope, nothing appears. It just shows the loading icon on my mouse for a split second and thats it.

Open Event Viewer (eventvwr.msc), go to Windows Logs -> Application, are there any entries marked as errors? If so, show me what they are please.

**Comment by fiull** [comment]
> > Nope, nothing appears. It just shows the loading icon on my mouse for a split second and thats it.
> 
> Open Event Viewer (eventvwr.msc), go to Windows Logs -> Application, are there any entries marked as errors? If so, show me what they are please.

Yes actually! it shows this entry every time i try to open it :

_Description: A .NET application failed.
Application: Bloxstrap-v2.5.4.exe
Path: C:\Users\???\Desktop\Bloxstrap-v2.5.4.exe
Message: A JSON parsing exception occurred in [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App\6.0.25\Microsoft.WindowsDesktop.App.runtimeconfig.json], offset 0 (line 1, column 1): The document is empty.
Invalid framework config.json [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App\6.0.25\Microsoft.WindowsDesktop.App.runtimeconfig.json]_

(hopefully i didin't share sensetive information on accident ^^')

**Comment by pizzaboxer** [comment]
Your .NET Runtime installation is screwed. [Reinstall it.](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.25-windows-x64-installer?cid=getdotnetcore). You can uninstall it through the installer, or by searching "desktop runtime" in the add/remove programs list.

**Comment by fiull** [comment]
> Your .NET Runtime installation is screwed. [Reinstall it.](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.25-windows-x64-installer?cid=getdotnetcore) You can uninstall it through the installer, or by searching "desktop runtime" in the add/remove programs list.

yup that fixed it, thanks alot! btw if i ever wanted to uninstall/disable bloxstrap, how do i do it? (not that im gonna do it lol)

**Comment by pizzaboxer** [comment]
Read the Wiki.

https://github.com/pizzaboxer/bloxstrap/wiki/Switching-between-Roblox-and-Bloxstrap
https://github.com/pizzaboxer/bloxstrap/wiki/Uninstalling-Bloxstrap
